sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -59,10 +59,12 @@ def __init__(
             proposal: Proposal distribution that is used to initialize the MCMC chain.
             theta_transform: Transformation that will be applied during sampling.
                 Allows to perform MCMC in unconstrained space.
-            method: Method used for MCMC sampling, one of `slice_np`, `slice`,
-                `hmc`, `nuts`. Currently defaults to `slice_np` for a custom numpy
-                implementation of slice sampling; select `hmc`, `nuts` or `slice` for
-                Pyro-based sampling.
+            method: Method used for MCMC sampling, one of `slice_np`,
+                `slice_np_vectorized`, `slice`, `hmc`, `nuts`. `slice_np` is a custom
+                numpy implementation of slice sampling. `slice_np_vectorized` is
+                identical to `slice_np`, but if `num_chains>1`, the chains are
+                vectorized (usually leads to faster sampling). The samplers `hmc`,
```

Comment:
maybe add that they are vectorized and sampled in parallel on a single core, instead of running chains one after the other sequentially.